spec/hardware/cpu.fmf

```diff
@@ -108,8 +114,13 @@ example:
     cpu:
         hyper-threading: true
 
+  - |
+    # Request a CPU with specified stepping.
+    cpu:
+        stepping: 9
```

Comment:
Unrelated changes, `stepping`?

Reply:
I see, stepping commits are included in this PR. You can edit this PR to change its base branch to the one providing stepping changes, that should remove stepping changes from this PR's diff.